RELEASE.md

```diff
@@ -1,10 +1,27 @@
 # Creating a Release
 
-[crates.io](https://crates.io/crates/ratatui) releases are automated via [GitHub actions](.github/workflows/cd.yml) and triggered by pushing a tag.
+[crates.io](https://crates.io/crates/ratatui) releases are automated via [GitHub
+actions](.github/workflows/cd.yml) and triggered by pushing a tag.
+
+1. Record a new demo gif. I found the best tool for this was ttyrec/ttygif. Asciinema handles block
```

Comment:
Maybe add a link to the mentioned projects?

Reply:
done